sklearn/impute.py

```diff
@@ -142,8 +135,7 @@ def fit(self, X, y=None):
             raise ValueError("Can only use these strategies: {0} "
                              " got strategy={1}".format(allowed_strategies,
                                                         self.strategy))
-
-        if self.axis not in [0, 1]:
+        if self.axis:
```

Comment:
This should be removed. The attribute no longer exists and the error is no longer relevant